docs/code/getting-started.md

```diff
@@ -108,53 +108,69 @@ Apps on elementary OS are organized into standardized directories contained in y
+                var app = new MyApp ();
+                return app.run (args);
+            }
         }
+        
+    Ready to test it out? Fire up your terminal and make sure you're in "~/Projects/gtk-hello/src". Then execute the following commands to compile and run your first Gtk+ app:
 
-3. Now, that we've initialized Gtk, we'll create a new window and declare a few properties of this window. You'll notice that most of these property names are pretty straightforward. Try to guess what each one does and we'll explain in a second. Type the following after the `Gtk.init` line, but before the last bracket:
+        ```bash
+        valac --pkg gtk+-3.0 gtk-hello.vala
+        ./gtk-hello
+        ```
 
-        var window = new Gtk.Window ();
-        window.title = "Hello World!";
-        window.set_border_width (12);
-        window.set_position (Gtk.WindowPosition.CENTER);
-        window.set_default_size (350, 70);
-        window.destroy.connect (Gtk.main_quit);
+    Did it work? If so, congratulations! If not, read over your source code again and look for errors. Also check the output of your terminal. Usually there is helpful output that will help you track down your mistake.
 
-        Gtk.main ();
-        return 0;
+3. Now, that we've created a new `Gtk.Application` with a `Gtk.ApplicationWindow` called `main_window`, let's declare a couple properties of this window. You'll notice that most of these property names are pretty straightforward. Try to guess what each one does and we'll explain in a second. Add the following properties to your `main_window`:
 
-    The first line creates a new `Gtk.Window` called "window". The second line sets the window title that you see at the top of the window. Next, we create a margin inside that window so that widgets don't bump up against the window's edge. Then we tell the window manager that we want to place this window in the center of the screen instead of in the default position (which is usually the top left). We also must give our window a default size so that is does not appear too small for the user to interact with it. Finally, we explain what to do with this process if the main window is closed; In our case, we want to quit.
+        var main_window = new Gtk.ApplicationWindow (this);
+        main_window.title = "Hello World!";
+        main_window.set_default_size (350, 70);
 
-4. Now that we've defined a nice window, let's put a button inside of it. After our window stuff (but before `Gtk.main` line), leave a new line and then type the following:
+        main_window.show_all ();
+
+    The first line creates a new `Gtk.ApplicationWindow` called `main_window`. The second line sets the window title that you see at the top of the window. We also must give our window a default size so that is does not appear too small for the user to interact with it. Compile and run your application again and see how it's changed from the first time.
```

Comment:
This paragraph describes what reader should see

> The second line sets the window title that you see at the top of the window. We also must give our window a default size so that is does not appear too small for the user to interact with it.